app/controllers/events_controller.rb

```diff
@@ -0,0 +1,39 @@
+class EventsController < ApplicationController
+  before_action :get_event , only: [:show, :new, :update]
+  before_action :authenticated!, only: [:update, :create]
+  def home
```

Comment:
What is home? Remember one thing we are not writing code only for ourselves. So it will be more meaning full in this case if we use index. Let's try to build that habit to follow railsy way.

Reply:
Ok, rest API way it is. Thanks for the review.  the option to create an event was always there on the users/show path. The username in the navbar is the link to the user page.

Odin asks to just authenticate with just a name. Let us know if an email is mandatory.

Reply:
> Odin asks to just authenticate with just a name. Let us know if an email is mandatory.

Nope. email is not mandatory. It's okay what you did. 🙂